backend/service/aws/dynamodb.go

```diff
@@ -2,23 +2,50 @@ package aws
 
 import (
 	"context"
+	"fmt"
 
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb"
 	"github.com/aws/aws-sdk-go-v2/service/dynamodb/types"
+	"go.uber.org/zap"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
 
 	dynamodbv1 "github.com/lyft/clutch/backend/api/aws/dynamodb/v1"
+	awsv1 "github.com/lyft/clutch/backend/api/config/service/aws/v1"
 )
 
+// defaults for the dynamodb settings config
+const (
+	AwsMaxRCU       = int64(40000)
+	AwsMaxWCU       = int64(40000)
+	SafeScaleFactor = float32(2.0)
```

Comment:
in general it's better to use untyped constants
```suggestion
	AwsMaxRCU       = 40000
	AwsMaxWCU       = 40000
	SafeScaleFactor = 2.0
```